tests/unit/test.linter.js

```diff
@@ -330,6 +330,7 @@ describe('Linter', () => {
     addonLinter.collector.addError = sinon.stub();
     addonLinter.print = sinon.stub();
     const expectedError = new Error('DuplicateZipEntry the zip has dupes!');
+    expectedError.name = 'DuplicateZipEntryError';
```

Comment:
Could we import `DuplicateZipEntryError` from addons-scanner-utils?

Reply:
@willdurand we can, but I think we need to update the addons-scanner-utils PR to export it (which would make it cleaner). I'm 100% of with making that change to that PR: